.github/html/congratulations.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
Meta description is a requirement. See example below
```
<meta name="description" content="Free Web tutorials">
```

Reply:
Thanks! Now fixed!